e2e/workspace/src/custom-layout.test.ts

```diff
@@ -63,5 +64,7 @@ describe('custom workspace layout', () => {
     expect(appBuildResults.stdout).toContain(`nx run ${expressApp}:build`);
 
     checkFilesExist(`dist/packages/${expressApp}/main.js`);
+
+    removeProject({ onlyOnCI: true });
```

Comment:
Only some of the tests have this?

Reply:
Yup, only workspace tests have this issue atm. Ideally, we don't want to remove the project in case someone wants to inspect it, so I don't want to introduce a mechanism to blindly delete old projects after each.